README.rst

```diff
@@ -61,7 +61,47 @@ and language *stability*
++-----------------------------------+----------------+----------------------------------+--------------------------+----------------------------------+
+| Awaiting shepherd recommendation  | 2 weeks        | Shepherd                         | Make recommendation      | Committee discussion             |
+|                                   |                |                                  +--------------------------+----------------------------------+
+|                                   |                |                                  | Send back for revision   | Community discussion             |
++-----------------------------------+----------------+----------------------------------+--------------------------+----------------------------------+
+| Committee discussion              | 4-5 weeks      | Shepherd                         | Accept                   | Accepted                         |
+|                                   |                |                                  +--------------------------+----------------------------------+
+|                                   |                |                                  | Reject                   | Rejected                         |
+|                                   |                |                                  +--------------------------+----------------------------------+
+|                                   |                |                                  | Send back for revision   | Community discussion             |
++-----------------------------------+----------------+----------------------------------+--------------------------+----------------------------------+
+| Accepted                          | 1-3 weeks      | Author                           | Do final corrections     | Merged                           |
+|                                   |                +----------------------------------+--------------------------+                                  |
+|                                   |                | Shepherd                         | Merge                    |                                  |
++-----------------------------------+----------------+----------------------------------+--------------------------+----------------------------------+
+| Accepted / Merged                 | Indefinite     | Anyone (including the author)    | Implement the proposal   | Implemented                      |
+|                                   |                | willing to implement it          |                          |                                  |
++-----------------------------------+----------------+----------------------------------+--------------------------+----------------------------------+
+
+Detailed list of stages
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The stage is identified by a GitHub label, which is identified in the following list.
 
 1. (No label.) The author drafts a proposal.
```

Comment:
I think we could merge points 1 and 2 so they correspond to the "proposal creation" stage in the table.